### DISCUSSION TITLE
Does the library support json file included in another json file?

### DISCUSSION THREAD

**Discussion by lanmozhileitan** [nlohmann/json#4158] [General]
My use case is to be able to parse nested json files, with one or multiple json files included in another json file. It can have arbitrary number of file inclusion depth(general cases are 1 or 2 levels down). If so, what's the syntax for json file inclusion?

I saw some nested json parsing support here: https://pypi.org/project/json-include/

**Comment by nlohmann** [comment]
No, this is not possible, but you can implement this easily with this library.
